pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/design/GenericClassCounterRule.java

```diff
@@ -51,6 +51,7 @@
  */
 public class GenericClassCounterRule extends AbstractJavaRule {
 
+    // Class is unused, properties won't be converted
```

Comment:
On first look, this rule might be use case for multi file analysis...

Reply:
Maybe... We should probably scrap the class and add its purpose somewhere on the roadmap

Reply:
Yes, no need to keep this class - we should just keep the idea/purpose of the rule.